src/main/java/vazkii/patchouli/client/book/template/BookTemplate.java

```diff
@@ -124,8 +124,14 @@ public void onDisplayed(BookPage page, GuiBookEntry parent, int left, int top) {
 	public void render(BookPage page, int mouseX, int mouseY, float pticks) { 
 		if(compiled)
 			components.forEach(c -> {
-				if(c.isVisible) 
+				if(c.isVisible) {	
```

Comment:
._.

Reply:
I have my formatter turned off so that it doesn't change every single line in every file I touch 😛